### PR TITLE
🚧 Punctuation reconstruction

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

**Comment by anuejn** [comment]
Two finds on this:
* I am not very convinced, that we should wait for server startup until the server is started. This unnescessarily makes audapolis feel slow. Also there is no handling of server startup errors.
* I would really like to have proper page back now because it is really unintuitive to go back to the landing page completely.

**Comment by pajowu** [comment]
I was unsure about the startup as well. However a lot of functionality needs the server. At the moment clicking the "Import"-Button too early leads to an unintuitive error (`x is undefined` or something like that) which sometimes (I couldn't reproduce when) persists until the application is reloaded. I could implement a waiting screen when clicking "Import" or "Settings", if you prefer that. However in the future we also want to use the server for alignment, so we even need it when opening an existing file (and correcting the transcript in it). Should we also present the user with a loading screen if they try to use this before the server fully started?

Regarding error handling: I think it's better to show the user a screen that leads them to notice that something is broken instead of them starting to use the app and then run into weird error messages.

**Comment by pajowu** [comment]
And regarding the back page: I think we should do that in another PR

File: app/src/pages/Landing.tsx [hunk 1]
```diff
@@ -23,10 +32,31 @@ const BottomRightContainer = styled.div`
...
+
+  return <AppContainer>{connected ? <LandingContent /> : <ConnectingContent />}</AppContainer>;
+}
+
+function ConnectingContent(): JSX.Element {
+  return (
+    <>
+      <TitleBar />
+      <MainCenterColumn>
+        <Heading>Connecting to server...</Heading>
```
Comment (anuejn) [inline app/src/pages/Landing.tsx:47]: I think the wording here is not really helpfull. I would rather prefer something like starting app up. Server sounds like we are leaking private info somewhere

File: app/src/tour/LanguageSettingsTour.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Tour } from '../components/Tour';
+import { Text } from 'evergreen-ui';
+
+const steps = [
+  {
+    target: '#transcription_table',
+    body: (
+      <Text>
+        At first you can download a so called "transcription models". These models will help the
```
Comment (anuejn) [inline app/src/tour/LanguageSettingsTour.tsx:10]: ⱥ

File: app/src/tour/TranscribeTour.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { Text } from 'evergreen-ui';
 const steps = [
   {
     target: '#model',
-    body: <Text>Here you should select the model you just downloaded...</Text>,
+    body: <Text>Here you should select the language you just downloaded...</Text>,
```
Comment (anuejn) [inline app/src/tour/TranscribeTour.tsx:8]: ,

File: server/app/transcribe.py [hunk 4]
```diff
@@ -1,3 +1,11 @@
+# TODO for punctuation
```
Comment (anuejn) [inline server/app/transcribe.py:1]: :wastebasket:

**Review by anuejn — APPROVED** [review]
seems fine to me. You are right it is better to handle other ux improvements in different prs

**Pull request merged**